pkgs/by-name/ne/neocities-cli/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, bundlerApp
+, bundlerUpdateScript
+}:
+
+bundlerApp {
+  pname = "neocities";
+  gemdir = ./.;
+  exes = [ "neocities" ];
+
+  passthru.updateScript = bundlerUpdateScript "neocities";
+
+  meta = with lib; {
+    description = "The Neocities Gem - A CLI and library for using the Neocities web site API.";
+    homepage = "https://github.com/neocities/neocities-ruby";
+    license = licenses.unfree;
```

Comment:
I just want to double check, if you are ok with sticking with this, there are implications of using unfree license, eg ofborg and hydra will not build such derivations.

Reply:
I'm ok with that, on the main github repo of the neocities cli there is no license so I guess I don't have the choice anyway.

Reply:
how about we merge this, but you follow up with upstream about setting some license ? then we can update it once upstream resolves it

Reply:
After digging a bit further in the source code of the project I found that in the Gemspec `spec.license = "MIT"`.
Should we consider that as enough to mark it with the MIT license, or do you still want me to contact upstream to get a definitive answer ?

Reply:
thats good enough for me, as thats what https://rubygems.org/ would use